transactions/update-contract/2023/may-3-rewards/README.md

```diff
@@ -0,0 +1,38 @@
+# Update FlowIDTableStaking Contract
+
+> May 3, 2023
+
+Upgrade to the `FlowEpoch` contract as per these pull requests:
+
+- https://github.com/onflow/flow-core-contracts/pull/352
+- https://github.com/onflow/flow-core-contracts/pull/365
+
+These include changes to enable automated staking rewards.
+
+## Transaction
+
+[upgrade_and_enable_rewards.cdc](./upgrade_and_enable_rewards.cdc)
+
+Used this to generate args:
+
+`cat "./FlowEpoch.cdc" | xxd -p | tr -d '\n'`
+
+Verified using:
+```
+$ cat arguments-update-contract-FlowEpoch-mainnet.json | jq '.[1] | .value' | xxd -r -p > /tmp/temp.txt
+$ diff /tmp/temp.txt FlowEpoch.cdc
+(Should produce no difference)
+```
+
+## Sequence of signing: 
+
+1. DapperLabs generates the Signature Request ID on the [site]() for the `upgrade_and_enable_rewards.cdc` transaction with the given args.
```

Comment:
For contract upgrades, the generated URL contains the encoded contract text which is very long, and the browser throws a "URL too long" error. 
Hence, for such upgrades, I copy past the args manually instead of generating a prepopulated URL link.
Will do the same this Wednesday.